libnest/src/package/error.rs

```diff
@@ -154,3 +154,23 @@ pub struct TagParseError(pub String);
 #[derive(Clone, Eq, PartialEq, Hash, Debug, Fail)]
 #[fail(display = "{}: invalid slot", 0)]
 pub struct SlotParseError(pub String);
+
+/// Type for errors related to the exploration of an NPF file
+#[derive(Debug)]
+pub struct NPFExplorationError {
+    inner: Context<NPFExplorationErrorKind>,
+}
+
+/// Kind for errors related to the exploration of an NPF file
+#[derive(Clone, Eq, PartialEq, Hash, Debug, Fail)]
+pub enum NPFExplorationErrorKind {
```

Comment:
What does this error adds in addition to `std::io::Error`? Why not return it instead?

Reply:
Oh I realised you probably did it to hide implementation details. 

Why not using a single member in this enum, named `io`, with the `#[cause]` serde attribut that would contain the `std::io::Error`? I think it would be more idiomatic.